requirements.txt

```
@@ -3,4 +3,9 @@ numpy
 networkx
 matplotlib
 cython
-scipy
\ No newline at end of file
+scipy
+pytest
+psutil
+joblib
+loky
+setuptools
```

```suggestion
scipy
psutil
joblib
loky
```
the [test requirements](https://github.com/duartegroup/autodE/blob/master/tests/requirements.txt) live in tests/ and I don't think should be a hard dependency if you only want to install autode. on setuptools: why is this needed? I've never had to install it. a windows thing?